src/SFA.DAS.EmployerIncentives.Domain/NewApprenticeIncentive/NewApprenticeIncentive.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+using SFA.DAS.EmployerIncentives.ValueObjects;
+
+namespace SFA.DAS.EmployerIncentives.Domain.NewApprenticeIncentive
+{
+    public class NewApprenticeIncentive : AggregateRoot
+    {
+        private static readonly DateTime EligibilityStartDate = new DateTime(2020, 8, 1);
+
+        public static bool IsApprenticeshipEligible(Apprenticeship apprenticeship)
```

Comment:
It doesn't look right to have static methods on domain object since the domain object isn't acting on it's own data and is acting just as a service.  Any reason these rules can't be in the apprenticeship object?